docker/Dockerfile.hipcc

```diff
@@ -0,0 +1,101 @@
+        ccache \
+        libbz2-dev \
+        libicu-dev \
+        python-dev \
+        autotools-dev \
+        libopenmpi-dev \
+        && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV PATH=/opt/rocm/bin:$PATH
+
+# Install CMake
+ENV CMAKE_DIR=/opt/cmake
+RUN CMAKE_VERSION=3.13.4 && \
+    CMAKE_KEY=2D2CEF1034921684 && \
+    CMAKE_URL=https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION} && \
+    CMAKE_SCRIPT=cmake-${CMAKE_VERSION}-Linux-x86_64.sh && \
+    CMAKE_SHA256=cmake-${CMAKE_VERSION}-SHA-256.txt && \
+    wget --quiet ${CMAKE_URL}/${CMAKE_SHA256} && \
+    wget --quiet ${CMAKE_URL}/${CMAKE_SHA256}.asc && \
+    wget --quiet ${CMAKE_URL}/${CMAKE_SCRIPT} && \
+    gpg --recv-keys ${CMAKE_KEY} && \
+    gpg --verify ${CMAKE_SHA256}.asc ${CMAKE_SHA256} && \
+    grep ${CMAKE_SCRIPT} ${CMAKE_SHA256} | sha256sum --check && \
+    mkdir -p ${CMAKE_DIR} && \
+    sh ${CMAKE_SCRIPT} --skip-license --prefix=${CMAKE_DIR} && \
+    rm ${CMAKE_SCRIPT}
+ENV PATH=${CMAKE_DIR}/bin:$PATH
+
+# Install Boost
+ENV BOOST_DIR=/opt/boost
+RUN BOOST_VERSION=1.72.0 && \
+    BOOST_VERSION_UNDERSCORE=$(echo "$BOOST_VERSION" | sed -e "s/\./_/g") && \
+    BOOST_KEY=379CE192D401AB61 && \
+    BOOST_URL=https://dl.bintray.com/boostorg/release/${BOOST_VERSION}/source && \
+    BOOST_ARCHIVE=boost_${BOOST_VERSION_UNDERSCORE}.tar.bz2 && \
+    wget --quiet ${BOOST_URL}/${BOOST_ARCHIVE} && \
+    wget --quiet ${BOOST_URL}/${BOOST_ARCHIVE}.asc && \
+    wget --quiet ${BOOST_URL}/${BOOST_ARCHIVE}.json && \
+    wget --quiet ${BOOST_URL}/${BOOST_ARCHIVE}.json.asc && \
+    gpg --recv-keys ${BOOST_KEY} && \
+    gpg --verify ${BOOST_ARCHIVE}.json.asc ${BOOST_ARCHIVE}.json && \
+    cat ${BOOST_ARCHIVE}.json | jq -r '. | .sha256 + "  " + .file' | sha256sum --check && \
+    gpg --verify ${BOOST_ARCHIVE}.asc ${BOOST_ARCHIVE} && \
+    mkdir -p boost && \
+    tar -xf ${BOOST_ARCHIVE} -C boost --strip-components=1 && \
+    cd boost && \
+    ./bootstrap.sh \
+        --prefix=${BOOST_DIR} \
+        && \
+    ./b2 -j${NPROCS} \
+        hardcode-dll-paths=true dll-path=${BOOST_DIR}/lib \
+        link=shared \
+        variant=release \
+        cxxflags=-w \
+        install \
+        && \
+    rm -rf boost*
+
+# Install Google Benchmark support library
+ENV BENCHMARK_DIR=/opt/benchmark
+RUN SCRATCH_DIR=/scratch && mkdir -p ${SCRATCH_DIR} && cd ${SCRATCH_DIR} && \
+    git clone https://github.com/google/benchmark.git -b v1.5.0 && \
+    cd benchmark && \
+    git clone https://github.com/google/googletest.git -b release-1.10.0 && \
+    mkdir build && cd build && \
+    cmake -D CMAKE_BUILD_TYPE=Release -D CMAKE_INSTALL_PREFIX=${BENCHMARK_DIR} .. && \
+    make -j${NPROCS} && make install && \
+    rm -rf ${SCRATCH_DIR}
+
+# Install Kokkos
+ARG KOKKOS_VERSION=0fdbdeda3d4d2cb00720627e46e3f2ecfb33e858
+ARG KOKKOS_OPTIONS="-DKokkos_ENABLE_HIP=ON -DKokkos_ENABLE_LIBDL=OFF -DKokkos_ENABLE_PROFILING=OFF -DKokkos_ARCH_VEGA906=ON"
```

Comment:
I thought the profiling issue was resolved

Reply:
Yes, enabling profiling should work.